package.json

```diff
@@ -10,7 +10,8 @@
     "react-component"
   ],
   "files": [
-    "build-lib"
+    "build-lib",
```

Comment:
With the addition of `src`, would `dist` be more standard here?

Reply:
Yes that is a good idea